src/setupWorker/glossary.ts

```diff
@@ -38,9 +38,9 @@ export interface ServiceWorkerIncomingRequest extends RequestWithoutMethods {
   id: string
 
   /**
-   * Text response body.
+   * Response body.
    */
-  body?: string
+  body?: ArrayBuffer
```

Comment:
Can this ever be undefined? 🤔

Reply:
We pass null intentionally sometimes .. I think this for handling that case.

Reply:
Then it should be 
```suggestion
  body: ArrayBuffer | null
```

In TypeScript null and undefined are treated differently. Could you please make this change to the type definition?